integration-test/env/D6608DD84293F393E2206151B84A4426A5C04B06.prv

```diff
@@ -0,0 +1,33 @@
+Bag Attributes
+    localKeyID: 01 00 00 00
+    Microsoft CSP Name: Microsoft Enhanced Cryptographic Provider v1.0
+Key Attributes
+    X509v3 Key Usage: 10
+-----BEGIN PRIVATE KEY-----
```

Comment:
Not really recommended to check in private keys, even if they are for tests. Consider writing a script that generates a keypair and encodes the relevant data for the tests.